app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -11,7 +11,7 @@ import android.util.Log
 
 class InstrumentService {
 
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
+    internal fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
```

Comment:
Updated the visibility modifiers per the suggestions here: https://kotlinlang.org/docs/reference/grammar.html#visibilityModifier